src/components/base/Button/index.js

```diff
@@ -221,7 +221,7 @@ class Button extends PureComponent<
     const onClickHandler = e => {
       if (onClick) {
         if (event) {
-          track(event, eventProperties)
+          // track(event, eventProperties)
```

Comment:
why the change?

Reply:
oh sorry, we needed to comment that for storybook to work, I will fix this